pkg/icingadb/overdue/sync.go

```diff
@@ -246,8 +250,10 @@ func (s Sync) updateOverdue(
 		op = s.redis.SRem
 	}
 
-	_, err := op(ctx, "icingadb:overdue:"+objectType, ids...).Result()
-	return err
+	cmd := op(ctx, "icingadb:overdue:"+objectType, ids...)
+	_, _ = cmd.Result() // Error is handled by icingaredis.WrapCmdErr().
```

Comment:
Maybe have a function that both calls `.Result()` and performs the error wrapping? Would avoid having to explain why there's no error handling here.